src/tools/kata-ctl/src/utils.rs

```diff
@@ -38,6 +40,120 @@ pub fn get_kernel_version(proc_version_file: &str) -> Result<String> {
     Ok(kernel_version)
 }
 
+const OS_RELEASE: &str = "/etc/os-release";
+
+//Clear Linux has a different path (for stateless support)
+const OS_RELEASE_CLR: &str = "/usr/lib/os-release";
+
+const UNKNOWN: &str = "unknown";
+
+pub fn get_distro_details(os_release: &str, os_release_clr: &str) -> Result<(String, String)> {
```

Comment:
Might be worth adding a comment referencing https://www.freedesktop.org/software/systemd/man/os-release.html.